.github/workflows/turborepo-release-step-2.yml

```diff
@@ -68,6 +72,16 @@ jobs:
       - uses: ./.github/actions/setup-go
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
+      - run: apt-get install -y build-essential musl-tools clang llvm gcc-aarch64-linux-gnu binutils-aarch64-linux-gnu
+      - uses: actions-rs/toolchain@v1
+      - run: |
+          rustup target add x86_64-pc-windows-gnu
+          rustup target add aarch64-unknown-linux-musl
+          rustup target add x86_64-unknown-linux-musl
+      - run: |
```

Comment:
This is a dumb workaround. We could a) figure out how to build these live or b) bake them into the buildcontainer. (could also address @NicholasLYang's point above)